CMakeLists.txt

```diff
@@ -35,6 +35,11 @@ set(ROCMLIR_BIN_DIR "${CMAKE_CURRENT_BINARY_DIR}/bin" CACHE PATH "")
 set(ROCMLIR_LIB_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib" CACHE PATH "")
 message(STATUS "ROCMLIR_BIN_DIR: ${ROCMLIR_BIN_DIR}")
 
+# BUILD_MIXR_TARGET is deprecated but still available
+if(BUILD_MIXR_TARGET)
+  set(BUILD_FAT_LIBROCKCOMPILER ON CACHE BOOL "Build static librockCompiler")
+-endif()
```

Comment:
Extra - here, probably causing your build failure.

Also, can we verify that this packages all the header files MIGraphX needs now?

Reply:
I can confirm that it packages everything that old packages included. But we might have packaged more files than what MIGraphX actually needs, but I'm not sure about that.